cip/Jenkinsfile

```diff
@@ -104,6 +104,22 @@ node('docker') {
             sh 'docker image save cosmotech/brewerysamplesolution_simulator:latest -o Generated/Artifact/brewerysamplesolution_simulator.tar'
           }
         }
+        stage('Generate SBOM'){
+          dir('src'){
+            sh 'curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b ~/.local/bin v0.66.0 && \
+                export PATH=$HOME/.local/bin/:$PATH && \
+                trivy image --format cyclonedx --output sbom.json cosmotech/brewerysamplesolution_simulator:latest'
+          }
+        }
+        stage('Publish SBOM to DT'){
+          dir('src'){
+            try {
+                dependencyTrackPublisher artifact: 'sbom.json', projectId: '77f1c795-86b0-48b5-9057-5c0dc397529b', synchronous: true
+            } catch (e) {
+                echo 'failed'
```

Comment:
Why catching and ignoring the error. Don't we want the job to fail if we can't submit the data?
I could see that we may want to treat this step as non-blocker regarding the job status, but we will never see a random "failed" log in the middle, so I rather be more aggressive and completely fail the job (so just removing the try/catch)